charts/bindplane/README.md

```diff
@@ -48,7 +48,7 @@ BindPlane OP is an open source observability pipeline.
 | backend.postgres.username | string | `""` | Username to use when connecting to Postgres. |
 | backend.type | string | `"bbolt"` | Backend to use for persistent storage. Available options are `bbolt`, and `postgres` (Enterprise). |
 | config.password | string | `""` | Password to use. Overrides `config.secret`. |
-| config.remote_url | string | `""` | URI used by agents to communicate with BindPlane using OpAMP. |
+| config.remote_url | string | `""` | URI used by agents to communicate with BindPlane using OpAMP. NOTE: This value will eventually be removed when support for older versions of BindPlane is removed. |
```

Comment:
We should specify which version of bindplane. I imagine the change over will happen in v1.15.0.